server/pkg/webserver/middleware/auth.go

```diff
@@ -0,0 +1,59 @@
+package middleware
+
+import (
+	"github.com/iris-contrib/middleware/jwt"
+	"github.com/kataras/iris/v12"
+)
+
+var (
+	mySecret     = []byte("login secret")
```

Comment:
Make it configurable?

Reply:
Do it in next version.

Reply:
Please add TODO here.

Reply:
> Please add TODO here.
// TODO: Make it configurable
	mySecret     = []byte("login secret") 
I hava added it.

Reply:
👌